newsfragments/41395.significant.rst

```diff
@@ -0,0 +1,12 @@
+**Breaking Change**
+
+The following deprecated functions, constants, and classes have been removed as part of the preparation for Airflow 3.0:
```

Comment:
```suggestion
The following deprecated functions, constants, and classes have been removed as part of Airflow 3.0:
```
Since this would be in the release note